pic-api/src/main/kotlin/com/mashup/pic/security/jwt/JwtManager.kt

```diff
@@ -62,7 +62,8 @@ class JwtManager(
 
     companion object {
         private const val CLAIM_USER_INFO_KEY = "user-info"
-        private const val ACCESS_TOKEN_EXPIRATION = 12 * 60 * 60 * 1000 // 12 hours
+//        private const val ACCESS_TOKEN_EXPIRATION = 12 * 60 * 60 * 1000 // 12 hours
+        private const val ACCESS_TOKEN_EXPIRATION = 30L * 24 * 60 * 60 * 1000 // 30 days
```

Comment:
develop에는 이유 없는 주석을 넣으면 안돼요!
주석을 지양해야 하지만 주석을 쓴 다면 주석을 단 이유를 적어주세요
// TODO: 를 사용해서
```suggestion
// TODO: Change access token expiration after developing
//        private const val ACCESS_TOKEN_EXPIRATION = 12 * 60 * 60 * 1000 // 12 hours
        private const val ACCESS_TOKEN_EXPIRATION = 30L * 24 * 60 * 60 * 1000 // 30 days
```